packages/js/docs/ts/classes/call.md

```diff
@@ -92,9 +90,9 @@ ___
 *Inherited from void*
 
 The `state` of the call.
-Usage with {@link State}:
```

Comment:
I didn't think this header made sense, since we don't publicly expose the State interface